calico/felix/test/test_config.py

```diff
@@ -660,3 +664,14 @@ def test_drop_valid(self):
             }
             config = load_config("felix_missing.cfg", host_dict=cfg_dict)
             self.assertEqual(config.ACTION_ON_DROP, value)
+
+    def test_interface_prefix(self):
+        cfg_dict = {"InterfacePrefix": "foo"}
+        config = load_config("felix_interface_prefix.cfg",
+                             host_dict=cfg_dict)
+        self.assertEqual(config.IFACE_PREFIX, ['foo'])
+
+        cfg_dict = {"InterfacePrefix": "foo,bar"}
+        config = load_config("felix_interface_prefix.cfg",
+                             host_dict=cfg_dict)
+        self.assertEqual(config.IFACE_PREFIX, ['foo', 'bar'])
```

Comment:
Ah, yes.  As @fasaxc wrote there, we do want to take that in principle, but just want to do it carefully so as to manage the merge conflicts. Let's continue discussing those details at #1107